sbc_switzerland/models/correspondence.py

```diff
@@ -189,8 +189,8 @@ def fix_priority(self):
                 'createdat': letter.scanned_date
             })
             text_id = tc.select_one(
-                "SELECT text_id FROM translation WHERE id = %s", [
-                    translation_id]).get('text_id')
+                "SELECT text_id FROM translation "
+                f"WHERE id = {translation_id}").get('text_id')
```

Comment:
This is wrong, the SQL queries are an exception where we should not use format, but continue to pass the parameters as exepected by the sql functions. We wisk SQL injections if we do otherwise.